modules/core/src/models/core.ts

```diff
@@ -209,22 +209,3 @@ export interface SVGRLoadedInterface {
      */
     default: string;
 }
-
-export interface LegacyModeInterface {
-    apiResourcesV1: boolean;
-    apiResourcesV2: boolean;
-    applicationListSystemApps: boolean;
-    applicationOIDCSubjectIdentifier: boolean;
```

Comment:
I think we might need this config for some IS users. @DonOmalVindula can confirm the final conclusion we came regarding this.

Reply:
As discussed offline, we can remove this config as well.